fairytale.md

```diff
@@ -15,4 +15,9 @@ The average Squonk is 7 meters tall and screams loud.
 >  What does the **world** he lives in **look like**?
 >  What happened between it and the *Drowsloks*?
 
-`Well... your turn now.`
+I think Squonks do not exist!
```

Comment:
Hi @Ozuitoku66  ! Could you please move your story above the line
```
>  Add some lines to the story. Can be anything. Inspiration?  
```
that is, around line 10?

More info will come later.